thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis/rootcause-table.ftl

```diff
@@ -1,18 +1,18 @@
 <div class="analysis-card padding-all top-buffer">
-  <h4 class="analysis-title bottom-buffer">[BETA] Root Cause Search
+  <h4 class="analysis-title bottom-buffer">[BETA] Root Cause Search</h4>
   <table class="table table-borderless rootcause__table">
     <thead>
     <tr>
       <td class="label-medium-semibold">Type</td>
-      <td class="label-medium-semibold">Details</td>
+      <td class="label-medium-semibold" style="max-width: 600px;">Details</td>
```

Comment:
can we have create a new css class and save the style in `thirdeye.css`)  perhaps `rootcause__details`.
Inline styling is usually viewed as bad practice.